zummon/src/commonMain/kotlin/companysurvey/Validation.kt

```diff
@@ -18,8 +18,8 @@ enum class Status {
 val validateContractedCapacity = Validator { survey: Survey ->
     val gridConnection = survey.getSingleGridConnection()
 
-    val contractedConnectionCapacity = gridConnection.electricity.getContractedConnectionCapacityKw()
-    val physicalConnectionCapacity = gridConnection.electricity.getPhysicalConnectionCapacityKw()
+    val contractedConnectionCapacity = gridConnection.electricity?.getContractedConnectionCapacityKw()
+    val physicalConnectionCapacity = gridConnection.electricity?.grootverbruik?.physicalCapacityKw
```

Comment:
With this code, in the case of kleinverbruik, it will say `Status.MISSING_DATA`. But this is not true. 

I think the previous version was good.